README.md

```diff
@@ -371,6 +375,68 @@ Invoke-ScriptAnalyzer -Path "C:\path\to\project" -Recurse
 
 Note that providing settings explicitly takes higher precedence over this implicit mode. Sample settings files are provided [here](https://github.com/PowerShell/PSScriptAnalyzer/tree/master/Engine/Settings).
 
+## Custom rules
```

Comment:
Please add an entry in the table of contents at the top of this Readme

Reply:
Done. Totally missed that.

Reply:
Thanks, I appreciate the table of contents has gone a bit out of sync generally in some areas, thanks for cleaning up the surrounding area :-)